.travis.yml

```diff
@@ -6,8 +6,8 @@ sudo: false
 perl6:
   - latest
 install:
-  - rakudobrew build-panda
-  - panda installdeps .
+  - rakudobrew build-zef
+  - zef --debug install .
```

Comment:
I didn't realize there was a default test script. This really goes deeper to a problem with Perl 6 travis set. It runs `prove` (and normal installation doesn't use `prove` for testing) and doesn't test the module is actually installable. So if your META is missing files in `provides` the current default travis test won't catch it.